File: plugins/uptime/uptime.1m.go
package main

import (
	"fmt"
	"os/exec"
	"strings"
)

func main() {
	cmd := exec.Command("crossbar", "uptime")
	out, err := cmd.Output()
	if err != nil {
		fmt.Println("⬆️ --\n---\nUnable to get uptime")
		return
	}

	uptime := strings.TrimSpace(string(out))
	if uptime == "" {
		fmt.Println("⬆️ --\n---\nUnable to get uptime")
		return
	}

	fmt.Printf("⬆️ %s\n", uptime)
	fmt.Println("---")
	fmt.Printf("System Uptime: %s\n", uptime)
	fmt.Println("Refresh | refresh=true")
}

File: plugins/uptime/uptime.1m.rs
use std::process::Command;
use std::str;

fn main() {
    let output = Command::new("crossbar")
        .arg("uptime")
        .output();

    match output {
        Ok(cmd_output) if cmd_output.status.success() => {
            let uptime = str::from_utf8(&cmd_output.stdout).unwrap_or("").trim();
            if !uptime.is_empty() {
                println!("⬆️ {}", uptime);
                println!("---");
                println!("System Uptime: {}", uptime);
                println!("Refresh | refresh=true");
            } else {
                println!("⬆️ --\n---\nUnable to get uptime");
            }
        }
        _ => {
            println!("⬆️ --\n---\nUnable to get uptime");
        }
    }
}